rbi/core/file.rbi

```diff
@@ -169,7 +169,7 @@ class File < IO
   # [`NotImplementedError`](https://docs.ruby-lang.org/en/2.6.0/NotImplementedError.html).
   sig do
     params(
-        file: BasicObject,
+        file: T.any(String, Pathname),
```

Comment:
this isn't backwards compatible.

please run ` ./sorbet/scripts/test-pr-on-pay-server 3246` in stripe monorepo